test/unit/pegjs-transform.js

```diff
@@ -1,16 +0,0 @@
-const pegjs = require( 'pegjs' );
```

Comment:
Don't we need this file anymore?

Reply:
Yes, some tests depend on the parser inside Gutenberg. I moved this to jest-preset-default in case someone ever uses the code that loads those files. Technically it all stays the same.

Reply:
Minor: Maybe, I'd have prefered this one to stay Gutenberg specific as it's not really a common thing to use `pegjs`

Reply:
I'm fine with leaving it here as suggested. I will update PR.